### PR TITLE
Improve error message around invalid testcase

### PR DESCRIPTION
**Pull request by tanin47**
When it's 404, 400, or 401, we print the error message and suggestion explicitly.

This solves #374

### CODE REVIEW

File: error/error/error.py [hunk 1]
```diff
@@ -58,21 +58,20 @@ def __init__(self):
         self.MESSAGE, self.EXIT_CODE)
 
 
-class ClusterfuzzAuthError(ExpectedException):
-  """An exception to deal with Clusterfuzz Authentication errors.
+class ClusterfuzzError(ExpectedException):
```
Comment (inferno-chromium) [inline error/error/error.py]: s/Clusterfuzz/ClusterFuzz everywhere in source

File: error/error/error.py [hunk 2]
```diff
@@ -235,3 +234,32 @@ def __init__(self, question):
...
+  def __init__(self, testcase_id):
+    super(InvalidTestcaseIdError, self).__init__(
+        self.MESSAGE.format(testcase_id=str(testcase_id)), self.EXIT_CODE)
+
+
+class UnauthorizedError(ExpectedException):
+  """An exception when the user cannot access the testcase."""
+
+  MESSAGE = (
+      "You aren't allow to access the testcase ID ({testcase_id}). "
```
Comment (inferno-chromium) [inline error/error/error.py]: s/allow/allowed

File: error/error/error.py (hunk 1 above)
Comment (tanin47) [inline error/error/error.py]: Done

File: error/error/error.py (hunk 2 above)
Comment (tanin47) [inline error/error/error.py]: Done

**Pull request merged**